### PR TITLE
Update changelog for v0.111.0

### PR DESCRIPTION
**Pull request by asreehari-splunk**
**Description:** In preparation for release v0.111.0

### CODE REVIEW

**Comment by atoulme** [comment]
Please take a look, it looks like the whole CHANGELOG is gone

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,67 @@
...
+- (Splunk) Remove ballast extension entirely from components  ([#5429](https://github.com/signalfx/splunk-otel-collector/pull/5429))
+
+- (Splunk) Deprecate jaegergrpc monitor ([#5428](https://github.com/signalfx/splunk-otel-collector/pull/5428))
+
+- (Splunk) Remove ballast extension entirely from components  ([#5429](https://github.com/signalfx/splunk-otel-collector/pull/5429))
+
+
+### 💡 Enhancements 💡
+
+- (Core) `mdatagen`: Generate documentation for components with resource attributes only ([#11242](https://github.com/open-telemetry/opentelemetry-collector/pull/11242))
```
[the remark this answers is not in the document]
Comment (dmitryax) [reply CHANGELOG.md:32]: Right. We don't ship mdatagen command tool. so this is not needed

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,67 @@
...
+
+- (Core) `confignet`: Add Profiles Marshaler to otlptext. ([#11161](https://github.com/open-telemetry/opentelemetry-collector/pull/11161))
+
+- (Contrib) `cmd/opampsupervisor:` Add passthrough logging for collector ([#35474](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33980))
```
Comment (dmitryax) [inline CHANGELOG.md:36]: Same here. Please remove since we don't ship the supervisor

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,67 @@
...
+
+- (Contrib) `receiver/awss3receiver:` Add ingest progress notifications via OpAMP ([#33980](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33980))
+
+- (Contrib) `exporter/clickhouse:` Add the ability to override default table names for all metric types ([#34251](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/34251))
+
+- (Contrib) `otelarrowexporter:` Adjust configuration based on experimental results ([#35478](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35478))
+
+- (Contrib) `processor/logdedup:` feat: add ottl condition to logdedup processor ([#35443](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35443))
```
Comment (dmitryax) [inline CHANGELOG.md:44]: These should be removed. We need only items for componets that are provided with our distro, see https://github.com/signalfx/splunk-otel-collector/blob/main/docs/components.md.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,67 @@
...
+
+- (Contrib) `receiver/azureblobreceiver:` support for default auth ([#35636](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35636))
+
+- (Contrib) bump go 1.22.8 ([#35693](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35693))
+
+- (Contrib) bump go 1.22.8 ([#35693](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35693))
```
Comment (dmitryax) [inline CHANGELOG.md:50]: Not relevant to end users

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,67 @@
...
+
+- (Contrib) update sapm-proto to 0.16.0 ([#35630](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35630))
+
+- (Splunk) Support filters and pdata being emitted from monitors ([#5379](https://github.com/signalfx/splunk-otel-collector/pull/5379))
```
Comment (dmitryax) [inline CHANGELOG.md:54]: This change has no effect yet, let's not mention it

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,67 @@
...
+
+- (Splunk) Update OpenTelemetry Dependencies to latest ([#5414](https://github.com/signalfx/splunk-otel-collector/pull/5414))
```
Comment (dmitryax) [inline CHANGELOG.md:56]: This is not needed. We already mention it in the line 7

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,67 @@
...
+
+- (Core) Remove deprecated funcs from processor package ([#11368](https://github.com/open-telemetry/opentelemetry-collector/pull/11368))
+
+- (Core) Deprecate and remove processorhelper.NewUnhealthyProcessorCreateSettings ([#11376](https://github.com/open-telemetry/opentelemetry-collector/pull/11376))
+
+- (Splunk) Remove ballast extension entirely from components  ([#5429](https://github.com/signalfx/splunk-otel-collector/pull/5429))
+
+- (Splunk) Deprecate jaegergrpc monitor ([#5428](https://github.com/signalfx/splunk-otel-collector/pull/5428))
+
+- (Splunk) Remove ballast extension entirely from components  ([#5429](https://github.com/signalfx/splunk-otel-collector/pull/5429))
```
Comment (dmitryax) [inline CHANGELOG.md:27]: Duplicate

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,67 @@
...
+- (Core) componentstatus: Remove deprecated functions ([#11363](https://github.com/open-telemetry/opentelemetry-collector/pull/11363))
+
+- (Core) service: remove deprecated types ([#11361](https://github.com/open-telemetry/opentelemetry-collector/pull/11361))
+
+
+### 🚩Deprecations 🚩
+
+- (Core) Remove deprecated funcs from processor package ([#11368](https://github.com/open-telemetry/opentelemetry-collector/pull/11368))
+
+- (Core) Deprecate and remove processorhelper.NewUnhealthyProcessorCreateSettings ([#11376](https://github.com/open-telemetry/opentelemetry-collector/pull/11376))
```
Comment (dmitryax) [inline CHANGELOG.md:21]: These are not a user-facing changes. Let's remove them

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,67 @@
...
+## Unreleased
+
+This Splunk OpenTelemetry Collector release includes changes from the opentelemetry-collector v0.111.0 and the opentelemetry-collector-contrib v0.111.0 releases where appropriate.
+
+### 🛑 Breaking changes 🛑
+
+
+- (Core) componentstatus: Remove deprecated functions ([#11363](https://github.com/open-telemetry/opentelemetry-collector/pull/11363))
+
+- (Core) service: remove deprecated types ([#11361](https://github.com/open-telemetry/opentelemetry-collector/pull/11361))
```
Comment (dmitryax) [inline CHANGELOG.md:14]: We don't need to include any changes from API changelog, only changes from End-user changelog https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.111.0

**Comment by dmitryax** [comment]
Some important items from https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.111.0 are missed, e.g.:
```
### 🛑 Breaking changes 🛑

- `signalfxexporter`: Do not exclude the metric `container.memory.working_set` (#35475)
- `sqlqueryreceiver`: Fail if value for log column in result set is missing, collect errors (#35068)
- `windowseventlogreceiver`: The 'raw' flag no longer suppresses rendering info. (#34720)
  Previously, this flag controlled two behaviors simultaneously:
    1. Whether or not the body of the log record was an XML string or structured object.
    2. Whether or not rendering info was resolved.
  A separate 'suppress_rendering_info' option now controls rendering info resolution.
  This is considered a breaking change because users setting only the 'raw' flag without also setting the
  new 'suppress_rendering_info' flag may see a performance decrease along with more detailed events.
  
### 🚩 Deprecations 🚩

- `sapmreceiver`: `access_token_passthrough` is deprecated (#35330)
  - "`access_token_passthrough` is deprecated."
  - "Please enable include_metadata in the receiver and add the following config to the batch processor:"
  ```yaml
  batch:
    metadata_keys: [X-Sf-Token]

### 💡 Enhancements 💡

- `receivercreator`: Validate endpoint's configuration before starting receivers (#33145)
- `hostmetricsreceiver`: Add ability to mute all errors (mainly due to access rights) coming from process scraper of the hostmetricsreceiver (#20435)
- `kubeletstats`: Introduce feature gate for deprecation of container.cpu.utilization, k8s.pod.cpu.utilization and k8s.node.cpu.utilization metrics (#35139)
- `pkg/ottl`: Add InsertXML Converter (#35436)
- `pkg/ottl`: Add GetXML Converter (#35462)
- `pkg/ottl`: Add ToKeyValueString Converter (#35334)
- `pkg/ottl`: Add RemoveXML Converter (#35301)
- `sqlserverreceiver`: Add computer name resource attribute to relevant metrics (#35040)
- `windowseventlogreceiver`: Add 'suppress_rendering_info' option. (#34720)
  When this flag is enabled, the receiver will not attempt to resolve rendering info. This can improve performance
  but comes at a cost of losing some details in the event log.
  
- `windowseventlogreceiver`: Move artificial "remote_server" field to 'attributes["server.address"]'. (#34720)

### 🧰 Bug fixes 🧰

- `sqlqueryreceiver`: Fix reprocessing of logs when tracking_column type is timestamp (#35194)
```

**Comment by dmitryax** [comment]
Let's keep the same formatting as before - no extra line between items

File: CHANGELOG.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (asreehari-splunk) [reply CHANGELOG.md:32]: ok. will remove

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,5 +1,51 @@
...
+## Unreleased
+
+This Splunk OpenTelemetry Collector release includes changes from the opentelemetry-collector v0.111.0 and the opentelemetry-collector-contrib v0.111.0 releases where appropriate.
+
+### 🛑 Breaking changes 🛑
+
+- (Contrib) signalfxexporter: Do not exclude the metric container.memory.working_set ([#35475](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35475))
+- (Contrib) sqlqueryreceiver: Fail if value for log column in result set is missing, collect errors ([#35068](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/35068))
+- (Contrib) windowseventlogreceiver: The 'raw' flag no longer suppresses rendering info. ([#34720](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/34720))
+- (Contrib)
```
Comment (atoulme) [inline CHANGELOG.md:14]:
```suggestion
```

File: CHANGELOG.md [hunk 11]
```diff
@@ -1,5 +1,51 @@
...
+
+### 🚩Deprecations 🚩
+
+- (Contrib) sapmreceiver: access_token_passthrough is deprecated ([#35330](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35330))
+- (Splunk) Remove ballast extension entirely from components  ([#5429](https://github.com/signalfx/splunk-otel-collector/pull/5429))
+- (Splunk) Deprecate jaegergrpc monitor ([#5428](https://github.com/signalfx/splunk-otel-collector/pull/5428))
+
+### 💡 Enhancements 💡
+
+- (Core) `confignet:` Add Profiles Marshaler to otlptext. ([#11161](https://github.com/open-telemetry/opentelemetry-collector/pull/11161))
```
Comment (atoulme) [inline CHANGELOG.md:24]:
```suggestion
- (Splunk) Initial release of standalone collector binaries for Linux (amd64/arm64) and Windows (amd64) with FIPS 140-2 support. These are experimental (alpha) binaries, and it is not suitable to use them in production environments. ([#5378](https://github.com/signalfx/splunk-otel-collector/pull/5378)):
  - `otelcol-fips_linux_<amd64|arm64>`: Built with [`GOEXPERIMENT=boringcrypto`](https://go.dev/src/crypto/internal/boring/README) and [`crypto/tls/fipsonly`](https://go.dev/src/crypto/tls/fipsonly/fipsonly.go).
  - `otelcol-fips_windows_amd64.exe`: Built with [`GOEXPERIMENT=cngcrypto`](https://github.com/microsoft/go/blob/microsoft/main/eng/doc/fips/README.md) and [`requirefips`](https://github.com/microsoft/go/blob/microsoft/main/eng/doc/fips/README.md#build-option-to-require-fips-mode) (the collector will panic if FIPS is not enabled on the Windows host).
  - Smart Agent components are not currently supported.
  - Download the binaries from the list of assets below.
- (Core) `confignet:` Add Profiles Marshaler to otlptext. ([#11161](https://github.com/open-telemetry/opentelemetry-collector/pull/11161))
```

**Comment by atoulme** [comment]
line 47 to 65 should be removed. They contain a duplicate 0.110.0 header and the unreleased notes that should be factored into the addition

**Pull request merged**